assets/js/components/Work/Tabs/Preservation/FileSetDropzone.jsx

```diff
@@ -103,7 +103,16 @@ function WorkTabsPreservationFileSetDropzone({
             <strong>File uploaded successfully</strong>
           </h4>
           <ul className="block">{acceptedFileItems}</ul>
-          <Button css={{ padding: "5px" }} onClick={handleRemoveFile}>
+          <Button
+            css={{
+              padding: "5px",
+              textDecoration: "underline",
+              textTransform: "unset",
+              color: "White",
```

Comment:
Thinking out loud here, but wondering how we can create App specific mixins to flavor our buttons or if we should just try and be more specific with our Variants in the design system.

Reply:
Yea, I hear ya.  In this case, it'd be ideal if this button could be `<Button isText....` and if it was inside a `Notification`, it'd automatically update to the text color and have a transparent background.